src/internet/podcasts/gpoddersearchpage.cpp

```diff
@@ -60,7 +60,7 @@ void GPodderSearchPage::SearchFinished(mygpo::PodcastListPtr list) {
 
   model()->clear();
 
-  for (mygpo::PodcastPtr gpo_podcast : list->list()) {
+  for (const mygpo::PodcastPtr& gpo_podcast : list->list()) {
```

Comment:
These pointer types should be treated as values, i.e., we always want to copy them.